src/app/ApplicationTemplate.Tests/Presentation/ChuckNorrisSearchPageViewModelShould.cs

```diff
@@ -88,7 +67,7 @@ public async Task UpdateSearchTermProperty()
 				.Should().Be(anything);
 		}
 
-		private void CheckForSearchMethodWhenQuotesAreLoading_SpecialConfiguration(IHostBuilder host)
+		private void CheckForSearchMethod_WhenQuotesAreLoading_SpecialConfiguration(IHostBuilder host)
```

Comment:
When the custom configuration is specific to 1 test, I suggest you use local function rather than a prefixed private method.